hack/testing/check-EFK-running.sh

```diff
@@ -292,4 +312,13 @@ for pod in $(oc get pods -l component=kibana-ops -o name); do
 done
 
 echo $TEST_DIVIDER
+echo "Checking ES contains common data model index templates"
+for pod in $(oc get pods -l component=es -o name | sed 's,pod/,,'); do
+  checkESContainsIndexTemplates "$pod" || EXIT_CODE=1
+done
+for pod in $(oc get pods -l component=es-ops -o name | sed 's,pod/,,'); do
```

Comment:
Better to explicitly check if using the ops cluster and skip this check, to avoid confusion in the future.
```
if [[ "$CLUSTER" == "true" ]]; then
    for pod in $(oc get pods -l component=es-ops -o name | sed 's,pod/,,'); do
...
```